transformations/auxiliary_negation_removal/transformation.py

```diff
@@ -0,0 +1,206 @@
+import spacy
+
+from initialize import spacy_nlp
+from interfaces.SentenceOperation import SentenceOperation
+from interfaces.SentencePairOperation import SentencePairOperation
+from tasks.TaskTypes import TaskType
+
+"""
+Auxiliary Negation Removal.
+    Remove auxiliary negations generating a sentence with oposite meaning.
+"""
+
+
+def auxiliary_negation_removal(sentence, nlp):
+
+    # Tokenize Sentence
+    doc = nlp(sentence)
+
+    # Initialize Variables
+    changed = False
+    new_sentence = []
+    supported_auxiliaries = [
+        "am",
+        "are",
+        "can",
+        "could",
+        "had",
+        "has",
+        "have",
+        "is",
+        "may",
+        "might",
+        "must",
+        "shall",
+        "should",
+        "was",
+        "were",
+        "will",
+        "would",
+    ]
+
+    # Evaluate Tokens
+    for token in doc:
+        # Add Token to Output Sentence
+        new_sentence.append(token)
+
+        # Initialize Variables
+        token_lowercased_lemma = token.lemma_.lower()
+
+        # Process Negations
+        if token_lowercased_lemma == "not" or token_lowercased_lemma == "n't":
```

Comment:
Do you also want to expand this to include negative adverbs like hardly, seldom, rarely, barely, never, etc.